examples/collinear_magnetism.jl

```diff
@@ -20,9 +22,9 @@ atoms = [Fe => [zeros(3)]];
 # To get the ground-state energy we use an LDA model and rather moderate
 # discretisation parameters.
 
-kgrid = [3, 3, 3]  # k-point grid (Regular Monkhorst-Pack grid)
-Ecut = 15          # kinetic energy cutoff in Hartree
-model_nospin = model_LDA(lattice, atoms, temperature=0.01)
+kgrid = [3, 3, 3]       # k-point grid (Regular Monkhorst-Pack grid)
+Ecut = 15 * u"hartree"  # kinetic energy cutoff
+model_nospin = model_LDA(lattice, atoms, temperature=3150u"K")
```

Comment:
I think I would add a comment here that this is about 0.01 Hartree, because using Kelvin here is not that usual (but it's good to show that it works!)

Reply:
Yes, I would also leave it in Hartree. I think we don't need to recall here that it can be specified with Unitfuls, it's well covered in the tutorial

Reply:
Fair enough. I agree it might be a bit too much here (sorry @berquist for asking you to do this earlier).